subhub/tests/unit/test_subhub.py

```diff
@@ -95,12 +105,10 @@ def test_customer_signup_server_stripe_error_with_params(
         content_type="application/json",
     )
 
-    data = json.loads(response.data)
+    loaded_data = json.loads(response.data)
```

Comment:
does `loaded_` prefix add much to this variable?  It was `data` before, that was ambiguous?

Reply:
Above there is a variable named data, I did not want to confuse the two variables.

Reply:
sounds good